css/style.css

```diff
@@ -16,4 +15,98 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+nav {
+  padding: 20px 40px 3px 60px;
+  background-color: rgb(247, 244, 241);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
```

Comment:
```suggestion
```
`flex-direction` is `row` by default, and `flex-wrap` is `nowrap` by default, so we can remove them. We usually remove code that doesn't have an effect